heron/proto/topology.proto

```diff
@@ -69,7 +75,10 @@ message Config {
     required string key = 1;
     // Only one of the below are set
     optional string value = 2;
-    optional bytes java_serialized_value = 3;
+    //optional bytes java_serialized_value = 3;
+    optional bytes serialized_value = 3;
+
+    required ConfigValueType type = 4;
```

Comment:
@billonahill I agree that it is quite possible that adding a new required field to config would break the existing topology, as the .defn file (containing this Config and StreamSchema) for already running topologies would not have this required field. I will consider making ConfigValueType optional.